scenes/player/player_controller.gd

```diff
@@ -0,0 +1,23 @@
+class_name PlayerController
+extends Node
+
+var axis: Vector2 = Vector2.ZERO
+var running: bool = false
+var interacting: bool = false
+
+
+func _process(_delta: float) -> void:
+	reset()
+
+	if Pause.is_paused(Pause.System.PLAYER_INPUT):
+		return
+
+	axis = Input.get_vector(&"ui_left", &"ui_right", &"ui_up", &"ui_down")
+	running = Input.is_action_pressed(&"running")
+	interacting = Input.is_action_just_released(&"ui_accept")
```

Comment:
I think that is better to keep the interacting in the `player_interaction.gd` script. Because a fighting player may not need NPC talking interaction. So adding a player_controller goes a bit against the modularity we want for the player. Please leave it out of this PR.